### ISSUE TITLE
Error: Fatal trace logic error - unexpected end event on CRA

### ISSUE THREAD

**Issue by FezVrasta** [ai/size-limit#129]
I have a create-react-app project, I'm trying to upgrade from size-limit@1 to 2.

I installed `size-limit@2` and `@size-limit/preset-app`.

My config is:
```

  "size-limit": [
    {
      "limit": "1.95MB",
      "path": "build/static/**/*.{js,css,svg,woff,woff2,png,ttf,html}"
    }
  ]
```

when I run `size-limit` I get:

```
$ size-limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added. Use emitter.setMaxListeners() to increase limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added. Use emitter.setMaxListeners() to increase limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGTERM listeners added. Use emitter.setMaxListeners() to increase limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGHUP listeners added. Use emitter.setMaxListeners() to increase limit
Error: Fatal trace logic error - unexpected end event
    at Function._createTasksFromEvents (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/lib/main-thread-tasks.js:111:17)
    at Function.getMainThreadTasks (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/lib/main-thread-tasks.js:238:35)
    at Function.compute (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/lib/main-thread-tasks.js:272:28)
    at Object.computeMainThreadTasks (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/index.js:13:36)
    at generateTasksReport (/Users/federicozivolo/Projects/xxx/web/node_modules/estimo/src/reporter.js:9:25)
error Command failed with exit code 1.
```

What's the right way to make it work?

I used to have `webpack: false` in my old config, but I removed it because otherwise it asked me to install `@size-limit/webpack`, which I think is not needed since I want it disabled?

**Comment by ai** [comment]
`preset-app` uses Puppeter to track the time to run JS. Seems like Puppeter needs more memory.

Is it happens on CI or on local computer? How much memory do you have?

**Comment by FezVrasta** [comment]
It's on my iMac, I got 24 GB of RAM :-(

**Comment by ai** [comment]
Hm. Seems like we have problem in Tracium. Can you open on Estimo? https://github.com/mbalabash/estimo

**Comment by FezVrasta** [comment]
Done

**Comment by FezVrasta** [comment]
It's now working with this configuration:

```
  "size-limit": [
    {
      "limit": "1.1MB",
      "path": "src/index.js"
    }
  ]
```

I guess I had to define the entry point of my app rather than pointing to my build files.

**Comment by ai** [comment]
How many files did you have? We can add warnings on nig number of files.

**Comment by ai** [comment]
Fixed https://github.com/ai/size-limit/commit/9e0122bb617c1407051998c9dc5ca34ff23e4705

**Comment by ai** [comment]
Released in 2.1.1

**Comment by FezVrasta** [comment]
Why ignoring non JS files should be a good preset? Most apps also have CSS files, jsons, svgs etc that sums up pretty quickly
